includes/kp-functions.php

```diff
@@ -487,3 +487,70 @@ function ( $feature ) {
 
 	return $unavailable_features;
 }
+
+/**
+ * Set the payment method title for a Klarna order.
+ *
+ * @param WC_Order $order WooCommerce order.
+ * @param array    $klarna_place_order_response The Klarna place order response.
+ * @return void
+ */
+function kp_set_payment_method_title( $order, $klarna_place_order_response ) {
+	$klarna_order_id = $klarna_place_order_response['order_id'];
+	$response        = KP_WC()->api->get_klarna_om_order( $order->get_billing_country(), $klarna_order_id );
+	if ( is_wp_error( $response ) || ! isset( $response['initial_payment_method']['description'] ) ) {
+		$klarna_method = $klarna_place_order_response['authorized_payment_method']['type'];
+		switch ( $klarna_method ) {
+			case 'invoice':
+				$klarna_method = 'Pay Later';
+				break;
+			case 'base_account':
+				$klarna_method = 'Slice It';
+				break;
+			case 'direct_debit':
+				$klarna_method = 'Direct Debit';
+				break;
+			default:
+				$klarna_method = null;
+		}
+	} else {
+		$klarna_method = $response['initial_payment_method']['description'];
+	}
+
+	$title = $order->get_payment_method_title();
+	$title = empty( $title ) ? 'Klarna' : $title;
+
+	if ( $klarna_method ) {
+		$order->update_meta_data( '_kp_payment_method', $klarna_method );
+		$title = "$title - $klarna_method";
+	}
+
+	$order->set_payment_method_title( $title );
+}
+
+/**
+ * Whether HPOS is enabled.
+ *
+ * @return bool
+ */
+function kp_is_hpos_enabled() {
+	if ( class_exists( OrderUtil::class ) ) {
+		return OrderUtil::custom_orders_table_usage_is_enabled();
+	}
+	return false;
+}
+
+/**
+ * Equivalent to WP's get_the_ID() with HPOS support.
+ *
+ * @return int|false|int the order ID if the ID exist, otherwise false or zero.
+ */
+   //phpcs:ignore
+  function kp_get_the_ID() {
+	$hpos_enabled = kp_is_hpos_enabled();
+	$order_id     = $hpos_enabled ? filter_input( INPUT_GET, 'id', FILTER_SANITIZE_NUMBER_INT ) : get_the_ID();
+	if ( empty( $order_id ) ) {
+		$order_id = filter_input( INPUT_POST, 'post', FILTER_SANITIZE_NUMBER_INT );
+	}
+	return empty( $order_id ) ? false : absint( $order_id );
+}
```

Comment:
Question: are these related to this fix? I don't see that the functions are used at all, so i'm wondering if you accidentally copied over some code from some other branch you where working on.